### PR TITLE
Docs: readability updates

### PR DESCRIPTION
**Pull request by kiwicopple**
- Self-hosting: 
  - cleans up docker/self hosting
- Realtime: https://docs-git-docs-updatesonaplane-supabase.vercel.app/docs/guides/realtime
  - Add a "landing page"
  - Turns the Quickstarts into "Usage" sections. This is better for Clippy to ingest (based on headings) and make the docs generally more readable. I left the quickstart for the "postgres changes" page can then extended it with usage
  - Moved the overview in each Quickstart to the Concepts page (because it's not so quick to read)
  - Added a page on Client Side throttling
  - Cleaned up the Quotas page - making it more readable

### CODE REVIEW

File: apps/docs/pages/guides/realtime/presence.mdx [hunk 1]
```diff
@@ -3,233 +3,92 @@ import StepHikeCompact from '~/components/StepHikeCompact'
 
 export const meta = {
   title: 'Presence',
-  subtitle: "Get up and running with Realtime's Presence feature",
-  breadcrumb: 'Realtime Presence Quickstart',
+  description: 'Share state between users wtih Realtime Presence.',
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/presence.mdx:6]:
```suggestion
  description: 'Share state between users with Realtime Presence.',
```

File: apps/docs/pages/guides/realtime/presence.mdx [hunk 2]
```diff
@@ -3,233 +3,92 @@ import StepHikeCompact from '~/components/StepHikeCompact'
...
+  subtitle: 'Share state between users wtih Realtime Presence.',
```
Comment (kiwicopple) [inline apps/docs/pages/guides/realtime/presence.mdx:7]:
```suggestion
  subtitle: 'Share state between users with Realtime Presence.',
```

**Comment by scarigor** [comment]
Hi! There is a [typo](https://github.com/supabase/supabase/blob/master/apps/docs/pages/guides/database/connecting-to-postgres.mdx#L18) in connecting-to-postgres.mdx guide. (**We provides**).

**Review by chasers — APPROVED** [review]

**Comment by kiwicopple** [comment]
thanks @scarigor !

**Review by gregnr — APPROVED** [review]

**Pull request merged**